packages/flutter/lib/src/widgets/form.dart

```diff
@@ -337,6 +338,7 @@ class FormField<T> extends StatefulWidget {
 class FormFieldState<T> extends State<FormField<T>> {
   T _value;
   String _errorText;
+  bool _valueChanged = false;
```

Comment:
per our previous discussion, would you rename this to be more descriptive? such as _hasInteractedByUser